synadm/cli/__init__.py

```diff
@@ -376,10 +385,15 @@ def root(ctx, verbose, batch, output, config_file):
     default value 'auto-retrieval' will try to discover the name using the
     method set by --server-discovery."""
 )
+@click.option(
+    "--ssl-verify", "-i",  is_flag=True, default=True, show_default=True,
+    help="""Whether or not SSL certificates should be verified. Set to False
+    to allow self-signed certifcates."""
+)
 @click.pass_obj
 def config_cmd(helper, user_, token, base_url, admin_path, matrix_path,
-               output, timeout, server_discovery, homeserver):
-    """ Modify synadm's configuration.
+               output, timeout, server_discovery, homeserver, ssl_verify):
+    """ Modify test synadm's configuration.
```

Comment:
test?

Reply:
Oh, that seems to be a leftover of some debugging by the OP or something. Corrected.